tests/infra/github.py

```diff
@@ -216,47 +240,59 @@ def install_release(self, tag):
         LOG.info("Unpacking debian package...")
         shutil.rmtree(install_directory, ignore_errors=True)
         install_cmd = ["dpkg-deb", "-R", download_path, install_directory]
-        assert infra.proc.ccall(*install_cmd).returncode == 0, "Installation failed"
+        assert (
+            subprocess.run(install_cmd, check=True).returncode == 0
```

Comment:
check=True already implies returncode == 0 and throws if not: https://docs.python.org/3/library/subprocess.html#subprocess.run, so the assert is unreachable.